klayout.dockerfile

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+
+FROM hdlc/build:build AS build
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+    g++ ruby-dev python3-dev qt5-default qt5-qmake qttools5-dev libqt5xmlpatterns5-dev qtmultimedia5-dev libqt5svg5-dev \
+ && apt-get autoclean && apt-get clean && apt-get -y autoremove \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/KLayout/klayout.git /tmp/klayout \
+ && mkdir -p /opt/klayout/usr/local/bin \
+ && cd /tmp/klayout && ./build.sh -bin /opt/klayout/usr/local/bin -rpath /usr/local/lib -option -j$(nproc) \
```

Comment:
I think we might be missing some option here. Even though Qt version and other features are autodetected, it would be safer to explicitly pass `-python -ruby -qt5`.

I'm not sure about `-bin`, which is an alias of `-prefix`. Typically, we want the prefix to be `/usr/local/` and the destdir to be `/opt/klayout`. That ensures the `/opt/*` is not hardcoded inside the binary/tool. In this case, it seems that `-bin` is used as destdir, indeed (https://github.com/KLayout/klayout/blob/master/build.sh#L565). Hence, it should be ok.

Reply:
Also my understanding it's OK as -rpath option is the only one influencing run-tme behaviour and that should reflect the real path without destdir.